droit-image.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>MiXTeeN</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link type="text/css" rel="stylesheet" href="//netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap.min.css">
+    <script src="//netdna.bootstrapcdn.com/bootstrap/3.0.0/js/bootstrap.min.js"></script>
+</head>
+
+<body itemscope itemtype="http://schema.org/Product">
+
+<div class="container ">
+<div class="row">
+	<div class="col-xs-12" style="margin-bottom: 2em;">
+		<img src="images/logo_mixteen_baseline.png" alt="Mix-TeeN" style="width:100%; text-align: center;"/>
```

Comment:
use the same typo for MiXTeeN and prefer `max-width:100%;`
`text-align: center;` should be on the `div` container